dashboard/src/components/ApiDocs/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Auth } from "shared/Auth";
+import SwaggerUI from "swagger-ui-react";
+import "swagger-ui-react/swagger-ui.css";
+
+// Request interface needed for avoiding type error in the requestInterceptor
+interface Request {
+  // tslint:disable-line
```

Comment:
why are you disabling the linter here?

Reply:
Because ts interfaces generally should start with `I`, but since I need it to override the type used in `requestInterceptor`.
Perhaps it is an issue with the `@types/swagger-ui-react module`.

Reply:
ah, that's weird, the swagger package is defining the Request interface but it's not exposing it.. Anyway, can you add a comment with that info?